public/login.html

```
@@ -8,140 +9,177 @@
+              class="w-full px-4 py-2.5 rounded-xl border border-slate-200 focus:outline-none focus:ring-2 focus:ring-brand/40 text-sm"
+              placeholder="choose_a_username" />
+          </div>
+          <div>
+            <label class="block text-sm font-medium text-slate-700 mb-1">Email</label>
+            <input id="r-email" type="email" autocomplete="email" required
+              class="w-full px-4 py-2.5 rounded-xl border border-slate-200 focus:outline-none focus:ring-2 focus:ring-brand/40 text-sm"
+              placeholder="you@example.com" />
+          </div>
+          <div>
+            <label class="block text-sm font-medium text-slate-700 mb-1">Password</label>
+            <input id="r-password" type="password" autocomplete="new-password" required minlength="6"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)